### PR TITLE
ffms2: Add package

### PR DESCRIPTION
**Pull request by oysteinkrog**

### CODE REVIEW

File: mingw-w64-ffms2/PKGBUILD [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  cd "${srcdir}/${packagedir}"
+}
+
+build() {
+  mkdir -p "${srcdir}/build-${MINGW_CHOST}"
+  cd "${srcdir}/build-${MINGW_CHOST}"
+
+  ../${packagedir}/configure \
+    --prefix=${MINGW_PREFIX} \
+    --host=mingw32 \
```
Comment (Alexpux) [inline mingw-w64-ffms2/PKGBUILD]: Is it not accept ````${MINGW_CHOST}````?

File: mingw-w64-ffms2/PKGBUILD [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+groups=("${MINGW_PACKAGE_PREFIX}")
+license=('MIT')
+options=('staticlibs' 'strip')
+depends=("${MINGW_PACKAGE_PREFIX}-ffmpeg")
+
+makedepends=("${MINGW_PACKAGE_PREFIX}-gcc" "${MINGW_PACKAGE_PREFIX}-pkg-config")
+source=("https://github.com/FFMS/ffms2/archive/${pkgver}.tar.gz")
+sha1sums=('bf09d8ed54e5aad86b352b4d335d29da016f2014')
+#source=("$_realname"::"git+http://github.com/InitialForce/FFmpeg#branch=master")
+#sha1sums=('SKIP')
```
Comment (Alexpux) [inline mingw-w64-ffms2/PKGBUILD]: Remove commented lines.

File: mingw-w64-ffms2/PKGBUILD [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+pkgver=2.20
+pkgrel=1
+pkgdesc="A cross-platform wrapper library around libav/FFmpeg"
+arch=('any')
+url="http://https://github.com/FFMS/ffms2/"
+groups=("${MINGW_PACKAGE_PREFIX}")
+license=('MIT')
+options=('staticlibs' 'strip')
+depends=("${MINGW_PACKAGE_PREFIX}-ffmpeg")
+
```
Comment (Alexpux) [inline mingw-w64-ffms2/PKGBUILD]: Remove empty line.

File: mingw-w64-ffms2/PKGBUILD [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+    --disable-debug \
+    --disable-static \
+    --enable-shared
+  make
+}
+
+package() {
+  cd "${srcdir}/build-${MINGW_CHOST}"
+  make DESTDIR="${pkgdir}" install
+}
```
Comment (Alexpux) [inline mingw-w64-ffms2/PKGBUILD]: Install also license file. See any other PKGBUILD to see how we do it.

File: mingw-w64-ffms2/PKGBUILD (hunk 1 above)
Comment (oysteinkrog) [inline mingw-w64-ffms2/PKGBUILD]: Hmm, I am not sure I understand?

File: mingw-w64-ffms2/PKGBUILD (hunk 4 above)
Comment (oysteinkrog) [inline mingw-w64-ffms2/PKGBUILD]: Ok

File: mingw-w64-ffms2/PKGBUILD (hunk 1 above)
Comment (Alexpux) [inline mingw-w64-ffms2/PKGBUILD]: Why you wrote ````--host=mingw32```` instead ````--host=${MINGW_CHOST}```` ?

**Comment by oysteinkrog** [comment]
Hmm, what should I do about the lack of a LICENSE file?

**Comment by Alexpux** [comment]
Does source archive contain LICENSE file?

**Comment by oysteinkrog** [comment]
No.

**Pull request merged**